### DISCUSSION TITLE
How do i downlaod ear trumpet with github as it doesnt work for microsoft store

### DISCUSSION THREAD

**Discussion by Cracked1l** [File-New-Project/EarTrumpet#972] [Q&A]
So im new to github, but when i go to the link on the eartrumpet website and click github, it leads to https://github.com/File-New-Project/EarTrumpet
but when i go to this i can only see code, add file, and go to file, but no download button plz help

**Comment by riverar** [comment]
The README on the front page (https://github.com/File-New-Project/EarTrumpet) answers that question. Hope that helps!
